app/src/components/searchBar/SearchBar.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+import { Input } from "tabler-react-2";
+
+export const SearchBar = ({ onSearch }) => {
+  const [query, setQuery] = useState("");
+
+  const handleChange = (value) => {
+    setQuery(value);
+    if (onSearch) {
+      onSearch(value);
+    }
+  };
+
+  return (
+    <div style={{ display: "flex", alignItems: "center", gap: "0.5rem" }}>
+      <Input
+        label="Search by name or email"
+        placeholder = "Ex: John Pork" //lol
```

Comment:
```suggestion
        placeholder = "Ex: Edward Fesser"
```

Sorry to be a party pooper

Reply:
all good lol.